src/cmd_argument.rs

```diff
@@ -268,3 +268,24 @@ impl CmdArgument for &Path {
         self.to_path_buf().prepare_config(config);
     }
 }
+
+/// See the [`CmdArgument`] implementation for [`Stdin`] below.
+pub struct Stdin<'a>(pub &'a str);
```

Comment:
I think the most general version of this would be:

```suggestion
pub struct Stdin<T: Into<String>>(pub T);
```

You could do:

```
pub struct Stdin<T: AsRef<str>>(pub T);
```

But with `T: Into<String>` you can avoid a clone when a `String` is passed.

Reply:
Nice, I'll use `Into<String>`.